### PR TITLE
IBM-19 script to run migrations executable by npm and make normal processes start with 'production' not altering the db schema

### PR DESCRIPTION
**Pull request by chrisns**

### CODE REVIEW

**Comment by chrisns** [comment]
@allandegnan please merge this when you make jenkins run `npm run migrations` before deploying

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "postinstall": "precommit install",
     "coverage-test": "multi='xunit=shippable/testresults/result.xml' istanbul cover _mocha",
     "coverage-report": "istanbul report cobertura --dir shippable/codecoverage/",
+    "migrations": "NODE_ENV=productionAlter node utils/run-migrations.js",
```
Comment (JoeChapman) [inline package.json]: I might be mistaken, but here you refer to the node env as `productionAlter `, but the docker file  refers to it as `production`, is that correct?

Comment (chrisns) [inline package.json]: @JoeChapman yes, the goal is to override what the dockerfile is set, so that the migrations can happen

**Comment by allandegnan** [comment]
add lewis to the team and then feel free to assign this to him.

**Pull request merged**